src/main/java/core/basesyntax/BinaryString.java

```diff
@@ -7,6 +7,14 @@
      * а возвращает String с представлением этого числа в двоичном виде.
      */
     public String toBinaryString(int value) {
-        return null;
+        if (value == 1 || value == 0) {
+            return "" + value;
```

Comment:
can we return result without string concatenation?

Reply:
sure